service/tbc/tbcfork_test.go

```diff
@@ -1268,6 +1280,17 @@ func TestFork(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	// Wait for tbc to insert all blocks
+	hasBlocks := false
```

Comment:
I think this can be simplified to (and for below), unless there is a reason not to:
```suggestion
	var hasBlocks bool
```

Go seems to generally prefer using the empty value vs doing it like this, but they both have the same behaviour. I usually prefer empty, but we mix both a lot so it's up to you :)